static/admin/config.yml

```diff
@@ -48,3 +67,8 @@ collections:
       file: "content/about-sidebar.md"
       fields:
         - {label: "Body", name: "body", widget: "markdown"}
+media_library:
+  name: "cloudinary"
```

Comment:
I couldn't figure out how to only enable Cloudinary for one field, so this would also store newsfeed images in Cloudinary, although I think that's OK!